console/console-init/ui/src/graphql-module/error-helper.ts

```diff
@@ -19,7 +19,7 @@ const onServerError = (error: any, dispactAction: any, states: any) => {
       hasNetworkError !== true &&
       dispactAction({
         type: types.SET_NETWORK_ERROR,
-        payload: true
+        payload: { statusCode: networkError.statusCode }
```

Comment:
add one check before getting statusCode, sometimes we will not have statusCode and it will cause the error on run time.
 payload: { statusCode: networkError && networkError.statusCode }

Reply:
Fixed.